packages/roosterjs-editor-dom/lib/list/VList.ts

```diff
@@ -31,6 +31,13 @@ import type {
     CompatibleNumberingListType,
 } from 'roosterjs-editor-types/lib/compatibleTypes';
 
+const numberDefinition = createNumberDefinition(
+    false /** isOptional */,
+    undefined /** value */,
+    0,
+    19
```

Comment:
Here is a trick:

Instead of hard code a number here, you can declare "MIN", "MAX" elements in the enum to have the same value with the last item, for example:
```ts
const enum MyEnum = {
 MIN = 1,
  A = 1,
  B = 2,
   ...
  Z = 26,
  Max = 26,
}
```
Then you can use the Max value to define the definition:
```ts
createNumberDefinition(false, undefined, 0, MyEnum.MIN, MyEnum.MAX);
```

So, if we want to add some new element to the enum someday, we know we also need to update the value of MAX, and no need to worry that we forget update the value here.